vars/processTemplate.groovy

```diff
@@ -44,3 +50,21 @@ def applyDefaults(provided=[:], templateParams) {
   setParam('RELEASE_VERSION') { Utils.shWithOutput(this, "git rev-list --count HEAD") }
   return params
 }
+
+ boolean isValidateTemlate(parameters=[[:]]){
+  def pass = true
+  def mandatParamList=["RELEASE_VERSION","SOURCE_REPOSITORY_URL","SOURCE_REPOSITORY_REF"]
+  def templateParam=[]
+  //to convert complex hashmap in list to simple list of params.
+  parameters.each{
```

Comment:
space between?  _readability and consistency_

`.each {`

Reply:
Done